src/store/reducer/RootReducer.ts

```diff
@@ -0,0 +1,60 @@
+import {createSlice, createAsyncThunk} from '@reduxjs/toolkit';
+import {Crypto} from 'interfaces';
+
+const initialState = {
+  cryptos: [] as Crypto[],
```

Comment:
you should use a type for the initialState

initialState = {
} as CryptoState

and in store/types.ts

type CryptoState {
   cryptos: Crypto[];
   loading: boolean;
   ......

}